<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_eq">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>20</y>
      <width>321</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_memmove">
       <property name="text">
        <string>memmove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strcpy">
       <property name="text">
        <string>strcpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strncpy">
       <property name="text">
        <string>strncpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_memcpy">
       <property name="text">
        <string>memcpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strcat">
       <property name="text">
        <string>strcat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strncat">
       <property name="text">
        <string>strncat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_memcmp">
       <property name="text">
        <string>memcmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strcmp">
       <property name="text">
        <string>strcmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strcoll">
       <property name="text">
        <string>strcoll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strncmp">
       <property name="text">
        <string>strncmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="strxfrm">
       <property name="text">
        <string>strxfrm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strtok">
       <property name="text">
        <string>strtok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strmemset">
       <property name="text">
        <string>strmemset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_strlen">
       <property name="text">
        <string>strlen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>str1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>str2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>n</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_str1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_str2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ans"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Your string:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Operator +=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pe">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Operator =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_equal">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEdit_str">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pe"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_equal"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushAlarm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Нажать, когда всё плохо</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
